src/routes/menu-router.js

```diff
@@ -21,6 +21,15 @@ router.route('/')
         });
     });
   })
+  .put((req, res) => {
+    menuController.publishMenu(req.body.date)
```

Comment:
Does this mean that any PUT /menu request is going to publish the menu? I think that we should be more specific, like:
PUT /menu/<id or date>/
{
  published: true
}

Also are you checking that the request is made by admin?